python/mujoco/usd/exporter.py

```diff
@@ -251,12 +251,10 @@ def _load_geom(self, geom: mujoco.MjvGeom):
 
     assert geom_name not in self.geom_names
 
-    texture_file = (
-        self.texture_files[
-            self.model.mat_texid[geom.matid][mujoco.mjTEXROLE_RGB]
-        ]
-        if geom.matid != -1
-        else None
+    geom_textures = (
```

Comment:
Since in downstream code you don't have different handling of None vs an empty list (you just check `if self.geom_textures`), then you should prefer to just use the empty list to indicate no textures. This means you don't have to have Optional in all your function signatures.

Also, this is a complicated expression. Can you break it up into:

```
if geom.matid == -1:
  geom_textures = []
else:
  geom_textures = [(self.texture_files[i], self.model.tex_type[i]) if i != -1 else None for i in self.model.mat_texid[geom.matid]]
```